modules/client-configuration/pages/clients-openeuler.adoc

```diff
@@ -0,0 +1,97 @@
+[[clients-openeuler]]
+= Registering {openeuler} Clients
+
+This section contains information about registering traditional and Salt clients running {openeuler} operating systems.
+
+
```

Comment:
openEuler is not supported in SUSE Manager (at least not yet), and it is not clear whether it'll be. Is this then still needed? I was told to remove all references in SUSE Manager documentation and only update it for Uyuni.

Reply:
Ok, in this case we probably need an ifeval in the `nav-` file to make sure that it will appear in Uyuni documentation only.  Maybe, this is already in and I missed it.